README.md

```diff
@@ -10,6 +10,12 @@ runs on. You just have to press enter once! Be careful.
 
 #### Obtain a USB Image
 
+###### Configure your server
+
+    $ vi ocemr.yml
```

Comment:
doesn't exist:

    + sudo cp -va ocemr.yml /tmp/11298.usb/ansible/
    cp: cannot stat 'ocemr.yml': No such file or directory

Reply:
Included the yml.